styles/GlobalStyle.js

```diff
@@ -1,6 +1,7 @@
 import { createGlobalStyle } from "styled-components";
 
 export default createGlobalStyle`
+ @import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap');
```

Comment:
Please make this part of a new User Story.
The font should be included with "next/font".
Just remove it for now and add it later in a safe way (data privacy => locally stored font).